generators/app/package-json.js

```diff
@@ -225,7 +225,7 @@ const packageJSON = (current, context) => {
   }
 
   if (context.css) {
-    result.scripts.pretest += ' postclean build:css';
+    result.scripts.pretest += ' clean build:css';
```

Comment:
small fix that I found converting a project, we may as well do clean and postclean.